modules/ztp-low-latency-for-distributed-units-dus.adoc

```diff
@@ -7,14 +7,23 @@
 
 = Low latency for distributed units (DUs)
 
-Low latency is an integral part of the development of 5G networks. Telecommunications networks require as little signal delay as possible to ensure quality of service in a variety of critical use cases.
+Low latency is an integral part of the development of 5G networks. Telecommunications networks
```

Comment:
Why did you add line breaks here? Please ensure your text editor is not introducing hard line breaks.

Reply:
Fixed for next iteration